Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -454,27 +454,37 @@ void CConfigMain::InitializeGUIValues()
 void CConfigMain::InitializeGUITooltips()
 {
 	// General - Basic
-	CPUThread->SetToolTip(_("This splits the Video and CPU threads, so they can be run on separate cores.\nCauses major speed improvements on PCs with more than one core, but can also cause occasional crashes/glitches."));
-	Framelimit->SetToolTip(_("This limits the game speed to the specified number of frames per second (full speed is 60 for NTSC and 50 for PAL)."));
+	CPUThread->SetToolTip(_("Splits the CPU and GPU threads so they can be run on separate cores.\nProvides major speed improvements on most modern PCs, but can cause occasional crashes/glitches."));
```

Comment:
You might want to change cores to threads because I believe dual core doesn't care about physical core threads v. virtual core threads. e.g. Can dual core run on single core processor hyper threaded to two cores? I'm not sure but correct me if I'm wrong.